src/main/java/com/gmail/yuramitryahin/dao/CinemaHallDaoImpl.java

```diff
@@ -51,4 +51,16 @@ public CinemaHall add(CinemaHall cinemaHall) {
             throw new DataProcessingException("Could not get all cinema halls ", e);
         }
     }
+
+    @Override
+    public CinemaHall get(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            return session.createQuery("select c from CinemaHall c where c.id=:id",
```

Comment:
`return session.get(CinemaHall.class, id);`
can made this more simple

Reply:
fixed